aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1000,7 +1006,13 @@ impl AptosVM {
             ))
         });
 
-        gas_meter.charge_intrinsic_gas_for_transaction(txn_data.transaction_size())?;
+        let multiplier = if txn_data.is_keyless() {
```

Comment:
So just to double check, keyless itself is already feature-gated, so we don't need to feature gate the gas charging here, right?

Reply:
Correct, keyless is feature gated & currently off on mainnet.